### ISSUE TITLE
Focal loss values

### ISSUE THREAD

**Issue by Gouiaa** [xingyizhou/CenterNet#234]
@xingyizhou I see that you did not add any epsilon value to the log inside the focal loss function. You did not get nan values for the focal loss, even at the begining!!?

**Comment by xingyizhou** [comment]
https://github.com/xingyizhou/CenterNet/blob/master/src/lib/models/utils.py#L9

**Comment by Amore-HDU** [comment]
@xingyizhou this change to sigmoid is very important!!!

**Comment by wuzuowuyou** [comment]
> this change to sigmoid is very important!!!

why you say that?

**Comment by wuzuowuyou** [comment]
> @xingyizhou this change to sigmoid is very important!!!
```
def _sigmoid(x):
  y = torch.clamp(x.sigmoid_(), min=1e-4, max=1-1e-4)
  return y
```
为什么加1e-4重要？
